Userland/Libraries/LibCore/ArgsParser.cpp

```diff
@@ -153,7 +154,11 @@ bool ArgsParser::parse(int argc, char* const* argv, FailureBehavior failure_beha
     }
 
     // We're done parsing! :)
-    // Now let's show help if requested.
+    // Now let's show version or help if requested.
+    if (m_show_version) {
+        outln(stdout, "git");
+        exit(0);
```

Comment:
Please only exit when `failure_behavior == FailureBehavior::Exit || failure_behavior == FailureBehavior::PrintUsageAndExit`, otherwise return `false`
(not doing this would make e.g. `cd --version` kill the shell)

Reply:
done, thanks.